### ISSUE TITLE
Wrong URLs in the kubernetes integration guide

### ISSUE THREAD

**Issue by bigTuxer** [projectcalico/calico#2272]
Hello,

i've found a not working urls in the kubernetes integration guide for the version 3.3.
[https://github.com/projectcalico/calico/blob/master/v3.3/getting-started/kubernetes/installation/integration.md](url)

On the point "Install the Calico plugins" you have to download the two binaries(calico and calico-ipam).
The two urls after the wget are not correct anymore, in the release directory you have these binaries  with the architecture in the filename. The urls in this document does not contain the architecture.

The filenames are:
- calico-amd64
- calico-ipam-amd64
- calico-arm64
- calico-ipam-arm64
- calico-ppc64le
- calico-ipam-ppc64le

The download urls point to:
- calico
- calico-ipam

## Expected Behavior
Download the CNI plugins

## Current Behavior
Get a http 404 error.

## Possible Solution
Correct the urls with one of the available architectures or add a placeholder and give a list of avalable architectures

## Steps to Reproduce (for bugs)
wget -N -P /opt/cni/bin https://github.com/projectcalico/cni-plugin/releases/download/v3.3.0/calico
wget -N -P /opt/cni/bin https://github.com/projectcalico/cni-plugin/releases/download/v3.3.0/calico-ipam

## Context
I've found these not working urls while i was following your integration guide.

Thanks for your great project :-)

**Comment by bigTuxer** [comment]
I found an other wrong link in this document, i'm not sure if i could append it to this Issue or create a new issue.

On "Install standard CNI loopback plugin" there is the following link:
[https://github.com/containernetworking/cni/releases/download/v0.6.0/cni-v0.6.0.tgz](url)

But this archive does not contain the loopback plugin, this is the correct archive:
[https://github.com/containernetworking/plugins/releases/download/v0.6.0/cni-plugins-amd64-v0.6.0.tgz](url)

**Comment by caseydavenport** [comment]
Yep, I think we should call out that the binaries are tagged with architectures and then given an example which saves the binary for amd64.

e.g. 

```
wget -O /opt/cni/bin/calico https://github.com/projectcalico/cni-plugin/releases/download/v3.3.0/calico-amd64
```

We should do the same for the cni/releases link as well.
